core/Plugin/ControllerAdmin.php

```diff
@@ -124,13 +124,24 @@ private static function notifyIfEAcceleratorIsUsed()
 
     private static function notifyWhenPhpVersionIsEOL()
     {
-        $notifyPhpIsEOL = Piwik::hasUserSuperUserAccess() && self::isPhpVersion53();
+        $isDeprecatedPhp = (self::isPhpVersion53() || self::isPhpVersion54());
+        $notifyPhpIsEOL = Piwik::hasUserSuperUserAccess() && $isDeprecatedPhp;
         if (!$notifyPhpIsEOL) {
             return;
         }
-        $message = Piwik::translate('General_WarningPiwikWillStopSupportingPHPVersion')
+
+        $nextRequiredMinimumPHP = '5.5';
+
+        $majorPhpVersion = null;
+        if(self::isPhpVersion53()) {
+            $majorPhpVersion = '5.3';
+        } elseif(self::isPhpVersion54()) {
+            $majorPhpVersion = '5.4';
+        }
```

Comment:
maybe put this up to the beginning and check for `$majorPhpVersion !== null && Piwik::hasUserSuperUserAccess()`? This way the outdated version doesn't have to be maintained in two places (before and after `notifyPhpIsEOL`).

Or maybe only check if it's at least PHP5.5 and if not show `phpversion()`?